### DISCUSSION TITLE
Technical Onboarding Email issue

### DISCUSSION THREAD

**Discussion by codeurzebs** [microsoft/SATechnicalOnboarding#137] [Questions]
Good morning MLSA_HELP! please I would like to know why I am being sent this email although I am following the program with my community at ICT University. I did everything as a prerequisite, whether it was on GitHub with my portfolio, whether it was organizing an event, and currently my community is following the challenge that I centralized in my community's platform ( https://portal.ngcodex.com ). pls help me to understand. Thank you
- [My community 600 members website](https://ngcodex.com)
- [Event preview](https://ngcodex.com/events/images)
- [Challenge preview](https://learn.microsoft.com/training/challenges?id=63317b09-fcae-417f-b9fb-67c77dc38593&WT.mc_id=cloudskillschallenge_63317b09-fcae-417f-b9fb-67c77dc38593&wt.mc_id=studentamb_288877)
- [Image Online and onsite preview](https://www.linkedin.com/posts/pilotezebs_msftstudentambassadors-activity-7167441895692955648-mZoV?utm_source=share&utm_medium=member_desktop)
- [Portfolio preview CI/CD with AZURE STATIC WEB](https://delightful-mud-04f7cec10.5.azurestaticapps.net)
- Another event coming soon with AI ...
- you can also check the statistic of my MLSA_contributor ID

By:
ZEBS HAUPUR
MLSA CAMEROUN
VICE PRESIDENT STUDENT UNION GOVERNMENT ICT University

![DFFEFE](https://github.com/microsoft/SATechnicalOnboarding/assets/120027323/af770aac-bff5-48dd-89ee-554153d03ed8)

**Comment by AnthonyByansi** [comment]
Hi @codeurzebs 
Please reach out to the SA-coordinator through teams to have you assisted on this matter. Utilize the Search option on teams to locate the SA-coordinator profile
![335517646-4f6a70f9-982a-4ff8-8aba-ed2f521a52bb](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/88d2dba8-08f8-48e5-a46a-cd52427048ad)

**Reply by codeurzebs** [reply]
Ok thanks you
